centos_build/Dockerfile.centos7.sdk

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2019-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
It this is a new file then just use 2021. If you have been working on it since 2019 then can leave it as it is.